Signal/src/view controllers/SignalsViewController.m

```diff
@@ -339,25 +339,32 @@ - (void)tableViewCellTappedDelete:(NSIndexPath *)indexPath {
                       preferredStyle:UIAlertControllerStyleAlert];
         [self presentViewController:removingFromGroup animated:YES completion:nil];
```

Comment:
The feedback will be that the row is gone.  =)

The only reason for the alert in the case of the group thread was as a *sort* of activity indicator because "leaving" the group requires a network request which induces delay.
 
Now, as of this pull request, If you've already left the group, there is no network request, thus no delay, thus no need for an activity indicator.